### ISSUE TITLE
Clean-up Figma and Wiki and Archive outdated documentation

### ISSUE THREAD

**Issue by mxajPrice** [hackforla/access-the-data#190]
## OVERVIEW
Our github wiki and Figma has a lot of outdated information that needs to be updated, cleaned up and organized.

## ACTIONS
- [x] Wiki Cleanup 
   - [x] Organize existing pages
   - [x] Go through each page and add context and clean up any outdated materials
   - [ ] Create pages that allow people to easily find guides/systems and product progress
- [ ] Figma Cleanup
   - [ ] Archive outdated figma pages
   - [x] Organize most updated/current material in its own individual space
       - [x] high fidelity wireframes
       - [ ] components
   - [ ] Clean up Design System 

### Resources/Instructions
examples of documentation of "how we do things" from other H4LA teams: 
- [TDM Calculator](https://github.com/hackforla/tdm-calculator/wiki/(Pilot-Page)-Usability-Testing)
- [Brigade Playbook ](https://github.com/codeforamerica/brigade-playbook/wiki/Process-overview)

current version of Access the Data's [research wiki](https://github.com/hackforla/access-the-data/wiki/Research)

**Comment by mxajPrice** [comment]
@KC243 We talked about this at the last planning week.  Let me know if you have any questions.

**Comment by mxajPrice** [comment]
@KC243 and @bkong, is this complete?

**Comment by bkong** [comment]
From my discussions with @KC243 during planning week, the [wiki](https://github.com/hackforla/access-the-data/wiki/ATD-Website-Overview) was updated to the state of the design system. Kevin did not have a chance to finish cleaning up Figma before leaving so that still needs to be done. My understanding of what this entails is:

1. Moving finalized designs to a separate Figma page to separate those from the older versions to avoid possible future confusion
2. Confirm that the finalized designs only use components from the 'Components' Figma page
3. Sync text between Hi Fi (Desktop) and Hi Fi (Mobile) designs

**Comment by lrchang2** [comment]
- Went through wiki and organized all pages based on if they fit "product" updates or were "team assets and tools". 
- Also updated persona page to reflect what information is outdated and our research pivot

**Comment by lrchang2** [comment]
@bkong and I:

- went through the figma and created a new page called "CURRENT Hi Fi Wireframe (Desktop and Mobile)" to put the most updated version of the wireframes. 
- consolidated the home, about us, and contact page for mobile and desktop
- Bailey will find tutorials/resources to help designers use components
- We determined that going forward, updates to the most current mobile and desktop version has to be approved before being applied 
- Developed a list of things that need to be cleaned up within the high fi wireframes that include: 
   - header component needs to be updated with new IA
   - need to create overlay of warning user about leaving the page for mobile (only has it for desktop)
   - updating the Contact Us Desktop version with the most updated header
   - checking that all pages are consistent on page dimensions, color use and content

**Comment by lrchang2** [comment]
- updated the Contact Us Desktop with most updated header and footer 
- updated Home and Contact Us page for desktop and mobile with updated copy from nov 5

**Comment by lrchang2** [comment]
finished organizing existing pages on the wiki. 
We need to create a guide for any new PM to add to the wiki

**Comment by mxajPrice** [comment]
I am going to close this out.  Anything here that still needs to be done can be put into a new issue.
